### PR TITLE
[S] The great space horror

### PR DESCRIPTION
**Pull request by Zergspower**
## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
add: ah shit
/:cl:

### CODE REVIEW

**Comment by Zergspower** [comment]
Closing for secrets

**Comment by OrionTheFox** [comment]
All in all I can't in good conscious agree with this addition. This is a redone Blob, which was removed for good reason. No matter how you paint it, a spreading mass with ghost-controlled minions isn't going to foster any roleplay environment, it'll be a mechanics rush to kill the issue before it occupies the whole round.
This has the added downside of being a mechanical "conversion antag" too, using the ghost-controlled minions to get even more ghost-controlled minions - which aside from the 'snowballing' that got conversion antags removed, is also against the entire point of the Antag Opt-In we just set up.

File: code/modules/unit_tests/simple_animal_freeze.dm [hunk 1]
```diff
@@ -165,6 +165,17 @@
...
+		/mob/living/simple_animal/hostile/fleshmind/hiborg,
+		/mob/living/simple_animal/hostile/fleshmind/himan,
+		/mob/living/simple_animal/hostile/fleshmind/treader,
+		/mob/living/simple_animal/hostile/fleshmind/phaser,
+		/mob/living/simple_animal/hostile/fleshmind/mechiver,
+		/mob/living/simple_animal/hostile/fleshmind/mauler_monkey,
+		/mob/living/simple_animal/hostile/fleshmind,
 
 		// DO NOT ADD NEW ENTRIES TO THIS LIST
 		// READ THE COMMENT ABOVE
```
Comment (OrionTheFox) [inline code/modules/unit_tests/simple_animal_freeze.dm:181]: As written here, we shouldn't be adding any new simple_animals. This is an obsolete system, these should be basic mobs.

**Comment by Zergspower** [comment]
> All in all I can't in good conscious agree with this addition. This is a redone Blob, which was removed for good reason. No matter how you paint it, a spreading mass with ghost-controlled minions isn't going to foster any roleplay environment, it'll be a mechanics rush to kill the issue before it occupies the whole round. This has the added downside of being a mechanical "conversion antag" too, using the ghost-controlled minions to get even more ghost-controlled minions - which aside from the 'snowballing' that got conversion antags removed, is also against the entire point of the Antag Opt-In we just set up.

this was meant to be closed but i forgot - it's not anything remotely close to what you'll think it's going to be

File: code/modules/unit_tests/simple_animal_freeze.dm (hunk 1 above)
Comment (Zergspower) [reply code/modules/unit_tests/simple_animal_freeze.dm:159]: it's in the process of being converted, it's why this PR is marked as [S] and has dozens of commits

**Comment by nikothedude** [comment]
this feels like a mad scientists lab

**Comment by Zergspower** [comment]
> this feels like a mad scientists lab

partly lmao - tryin to get this last CI clear so i can run the event and establish some new ruins

**Review by SomeRandomOwl — APPROVED** [review]

**Pull request merged**